core/templates/dev/head/pages/exploration_editor/ChangeListService.js

```diff
@@ -0,0 +1,229 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+// A service that maintains a provisional list of changes to be committed to
```

Comment:
write @fileoverview at the beginning (like [here](https://github.com/YimingPan/oppia/blob/fb4ad4c12d4bb60a329abdea8cb459e8a04d72aa/core/templates/dev/head/pages/exploration_editor/EditorServices.js#L15)).  Also add a new line after fileoverview.

Reply:
Got it!

Reply:
I've pushed a new commit to fix this. PL check it out.